src/index.html

```diff
@@ -17,16 +17,27 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
```

Comment:
Ensure that the indentation is consistent. The `form` element (line 20) and its child `input` element (line 24) should have a 2-space indentation from their parent elements.